pennylane/templates/embeddings/basis.py

```diff
@@ -87,20 +87,44 @@ def __init__(self, features, wires, do_queue=True, id=None):
         if not set(features).issubset({0, 1}):
             raise ValueError(f"Basis state must only consist of 0s and 1s; got {features}")
 
-        super().__init__(features, wires=wires, do_queue=do_queue, id=id)
+        self._hyperparameters = {"basis_state": features}
+
+        super().__init__(wires=wires, do_queue=do_queue, id=id)
```

Comment:
I'm making `basis_state` a hyperparameter, because it is not trainable. That means that this gate does not take any parameters...